ui/webapp.py

```diff
@@ -45,12 +45,19 @@ def reset_results(*args):
         state.raw_json = None
 
     # Title
-    st.write("# Haystack Demo")
+    st.write("# Haystack Demo - Countries Facts")
```

Comment:
Let try to phrase this a bit more exciting. How about something like "Haystack Demo: Explore the world"